app/controllers/application_controller.rb

```diff
@@ -20,7 +19,7 @@ def current_user
       user = nil
       User.current_user = nil
       if token.valid?
-        user_id = token.data['user_id']
+        user_id = token.data[0]['user_id']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.